bundles/org.openhab.binding.flicbutton/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -7,6 +7,7 @@
 	<bridge-type id="flicd-bridge">
 		<label>FlicButton Bridge</label>
 		<description>This bridge represents a running instance of the fliclib-linux-hci server (flicd).</description>
+		<semantic-equipment-tag>NetworkAppliance</semantic-equipment-tag>
```

Comment:
From the documentation this seems to me like a local service:
```suggestion
		<semantic-equipment-tag>WebService</semantic-equipment-tag>
```

However, I'm not even sure if it qualifies as a "web" service, or just a [service](https://github.com/50ButtonsEach/fliclib-linux-hci)?

Reply:
We don't have a tag for `Service` so you choose please.

Reply:
PS it is something running on Linux on a local machine; and that machine is either a `NetworkAppliance` or maybe just a `Computer`..